luigi/contrib/ecs.py

```diff
@@ -133,6 +132,7 @@ class ECSTask(luigi.Task):
 
     task_def_arn = luigi.Parameter(default=None)
     task_def = luigi.Parameter(default=None)
+    cluster = luigi.Parameter(default=None)
```

Comment:
Could you update the above docstring to include this parameter and a description? (I don't use ECS, so perhaps this is obvious and unnecessary)